client/templates/articles.coffee

```diff
@@ -11,13 +11,46 @@ Template.articles.helpers
   diseases: ->
     Template.instance().diseases.get()
 
+  articlesReady: ->
+    Template.instance().articlesReady.get()
+
 Template.articles.events
-  "change #diseases": (event) ->
+
+  "click .block": (event, template) ->
+    $(".block").removeClass("current")
+    $(event.target).addClass("current")
+
+  "change #diseases": (event, template) =>
+    $(".block").removeClass("current")
+    $(".month").addClass("current")
+    Modal.show("loadingModal")
+    Meteor.call 'getDiseaseInfoByMonth', $("#diseases").val(), (err, result) =>
```

Comment:
So `$("#diseases").val()` is better to be changed to `template.$("#diseases").val()` to save the browser from looping too much through the DOM tree.